File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
     Map<Integer,Integer> s = new HashMap<>();
     int opp;
     for(int i=0;i<nums.length;i++){
        opp=target-nums[i];
        if(s.containsKey(opp))
        {
           return new int[]{s.get(opp), i};
        }else{
            s.put(nums[i],i);
        }

     }
     return nums;     
    }
}

File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int i = 0;
        while (i < nums.size()) {
            for (int j = i + 1; j < nums.size(); j++) {
                if (target - nums[j] == nums[i]) {
                    return {j, i};
                }
            }
            i++;
        }
        return {};
    }
};
